packages/ui/v2/modules/booker/DatePicker.tsx

```diff
@@ -188,8 +189,18 @@ const DatePicker = ({
           </div>
         ))}
       </div>
-      <div className="grid grid-cols-7 gap-1 text-center">
+      <div className="relative grid grid-cols-7 gap-1 text-center">
         <Days weekStart={weekStart} selected={selected} {...passThroughProps} browsingDate={browsingDate} />
+
+        {/* TODO: only show when no slots available */}
```

Comment:
hey team, I need help here. any ideas how to best check for isEmpty && ?

Reply:
@emrysal maybe you have a quick thought, don't wanna distract you from Date Overwrite tho

Reply:
we can move this container into <Days>

Reply:
that worked!